adapter/i_o.py

```diff
@@ -854,3 +856,44 @@ def process_column_labels(self, list_of_labels):
         ]
 
         return list_of_cleaned_labels
+
+
+def to_pickle(obj, out_path):
```

Comment:
If there's desire to add pickled dict of dataframes as types that could be read by the IO class (e.g. dict_of_dfs = IO(path/to/pickled/dfs).get_tables(path/to/pickled_dfs)), I'd suggest adding calls to these methods in the IO class. 

If that's not planned fully yet we can keep these separate methods as-is

If this hasn't been planned yet then these